config/filter.d/cyrus-imap.conf

```diff
@@ -13,7 +13,7 @@ before = common.conf
 
 _daemon = (?:cyrus/)?(?:imap(d|s)?|pop3(d|s)?)
 
-failregex = ^%(__prefix_line)sbadlogin: \S+ ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
+failregex = ^%(__prefix_line)sbadlogin: (\S+)? ?\[<HOST>\] \S+ .*?\[?SASL\(-13\): (authentication failure|user not found): .*\]?$
```

Comment:
1. alternatively just `\S* ?` (but I like sebres's more explicit one I guess... I don't care if group is capturing or not and would value more "readability" of the regex, or should we care about capturing one?
2. Please do not remove any trailing anchoring unless it is indeed greedy catch-all.  Here ideally it should get that lookback to demand trailing `\]`  if leading one was present